third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -242,6 +254,10 @@ func resourceBigtableInstanceUpdate(d *schema.ResourceData, meta interface{}) er
 	}
 	conf.DisplayName = displayName.(string)
 
+	if _, ok := d.GetOk("labels"); ok {
```

Comment:
I think this if statement needs to be removed to allow removing all labels. See this test failure:
```
------- Stdout: -------
=== RUN   TestAccBigtableInstance_cluster
=== PAUSE TestAccBigtableInstance_cluster
=== CONT  TestAccBigtableInstance_cluster
TestAccBigtableInstance_cluster: testing.go:674: Step 7 error: After applying this step, the plan was not empty:
DIFF:
UPDATE: google_bigtable_instance.instance
cluster.#:              "4" => "4"
cluster.0.cluster_id:   "tf-test-14k76bhjh4-a" => "tf-test-14k76bhjh4-a"
cluster.0.num_nodes:    "5" => "5"
cluster.0.storage_type: "HDD" => "HDD"
cluster.0.zone:         "us-central1-a" => "us-central1-a"
cluster.1.cluster_id:   "tf-test-14k76bhjh4-c" => "tf-test-14k76bhjh4-c"
cluster.1.num_nodes:    "5" => "5"
cluster.1.storage_type: "HDD" => "HDD"
cluster.1.zone:         "us-central1-c" => "us-central1-c"
cluster.2.cluster_id:   "tf-test-14k76bhjh4-b" => "tf-test-14k76bhjh4-b"
cluster.2.num_nodes:    "5" => "5"
cluster.2.storage_type: "HDD" => "HDD"
cluster.2.zone:         "us-central1-b" => "us-central1-b"
cluster.3.cluster_id:   "tf-test-14k76bhjh4-d" => "tf-test-14k76bhjh4-d"
cluster.3.num_nodes:    "5" => "5"
cluster.3.storage_type: "HDD" => "HDD"
cluster.3.zone:         "us-central1-f" => "us-central1-f"
deletion_protection:    "false" => "false"
display_name:           "tf-test-14k76bhjh4" => "tf-test-14k76bhjh4"
id:                     "projects/ci-test-project-188019/instances/tf-test-14k76bhjh4" => "projects/ci-test-project-188019/instances/tf-test-14k76bhjh4"
instance_type:          "PRODUCTION" => "PRODUCTION"
labels.env:             "default" => ""
name:                   "tf-test-14k76bhjh4" => "tf-test-14k76bhjh4"
project:                "ci-test-project-188019" => "ci-test-project-188019"
STATE:
google_bigtable_instance.instance:
ID = projects/ci-test-project-188019/instances/tf-test-14k76bhjh4
provider = provider.google
cluster.# = 4
cluster.0.cluster_id = tf-test-14k76bhjh4-a
cluster.0.num_nodes = 5
cluster.0.storage_type = HDD
cluster.0.zone = us-central1-a
cluster.1.cluster_id = tf-test-14k76bhjh4-c
cluster.1.num_nodes = 5
cluster.1.storage_type = HDD
cluster.1.zone = us-central1-c
cluster.2.cluster_id = tf-test-14k76bhjh4-b
cluster.2.num_nodes = 5
cluster.2.storage_type = HDD
cluster.2.zone = us-central1-b
cluster.3.cluster_id = tf-test-14k76bhjh4-d
cluster.3.num_nodes = 5
cluster.3.storage_type = HDD
cluster.3.zone = us-central1-f
deletion_protection = false
display_name = tf-test-14k76bhjh4
instance_type = PRODUCTION
labels.env = default
name = tf-test-14k76bhjh4
project = ci-test-project-188019
--- FAIL: TestAccBigtableInstance_cluster (51.96s)
FAIL
```

Reply:
@danawillow  I've changed the if-GetOk clause to if-HasChange. I reckon that should have fixed the update call.

```
if d.HasChange("labels") {
	conf.Labels = expandLabels(d)
}
```